core/shared/platform/include/platform_api_extension.h

```diff
@@ -195,6 +195,93 @@ os_cond_signal(korp_cond *cond);
 int
 os_cond_broadcast(korp_cond *cond);
 
+/**
+ * Creates a new POSIX semaphore or opens an existing
+ * semaphore.  The semaphore is identified by name.  For details of
+ * the construction of name
```

Comment:
"For details of the construction of name" this statement seems incomplete

Reply:
Yes, docs added.
Most description of these API is from https://man7.org/linux/man-pages/man3/sem_open.3.html